src/appleseed/renderer/kernel/rendering/pixelrendererbase.h

```diff
@@ -32,20 +32,26 @@
 
 // appleseed.renderer headers.
 #include "renderer/kernel/rendering/ipixelrenderer.h"
+#include "renderer/modeling/frame/frame.h"
```

Comment:
There's already a `Frame` forward declaration so this seems redundant.

Reply:
Yes, my bad